cmake/SetupSpheral.cmake

```diff
@@ -140,6 +148,10 @@ set_property(GLOBAL PROPERTY SPHERAL_CXX_DEPENDS "${SPHERAL_CXX_DEPENDS}")
 #-------------------------------------------------------------------------------
 # Prepare to build the src
 #-------------------------------------------------------------------------------
+configure_file(${SPHERAL_ROOT_DIR}/src/config.hh.in
```

Comment:
I think this is happening twice now? Once here and once in src/CMakeLists.txt

Reply:
I moved it here. I don't see it in `src/CMakeLists.txt` anymore.

Reply:
https://github.com/LLNL/spheral/blob/d4cada370c1b4dcf8b225ca0a775b300548ba18f/src/CMakeLists.txt#L75
Isn't this it?

Reply:
Yep, that change wasn't committed, good catch